### PR TITLE
[DE] Translation Trading

### PR DESCRIPTION
**Pull request by DerSchlupf**
German Translation Trading-Section:
image

### CODE REVIEW

**Comment by DerSchlupf** [comment]
I will fix some problems (less text in the left small box; shorter text in order to move "handeln" in the left column, etc.

**Comment by qwrtln** [comment]
Yeah, you can clearly shave off at least 4 lines, if you get rid of the [runts](https://en.wikipedia.org/wiki/Widows_and_orphans).

File: translations/trading.tex/de.po [hunk 1]
```diff
@@ -78,6 +79,21 @@ msgid ""
...
+"\\begin{itemize}\n"
+"  \\item \\pagetarget{Trading}{Handle}\\index{Handeln} beliebig viele Ressourcen gemäß der Handelskurse laut \\hyperlink{Trade Table}{Tabelle auf dem rückseitigen Cover},\n"
+"  \\item \\textbf{Entferne genau eine Karte} aus deiner Hand und erhalte 1 \\svg{gold},\n"
+"    \\note{5}{Alle Startkarten (Sonderfähigkeiten, Statistikarten, die Startfähigkeit und Magische Pfeile) können im Handelsposten \\textbf{nicht entfernt werden}.}\\par\n"
+"  \\item Kaufe genau eine \\pagelink{War Machines}{Kriegsmaschine} (benötigt Schutzwall-Erweiterung).\n"
+"\\end{itemize}\n"
+"In \\textbf{Allianz-} und \\textbf{kooperativen} Szenarien gelten für die Spieler folgende weiteren Regeln zum Handeln von Ressourcen und Karten:\n"
+"\\begin{itemize}\n"
+"  \\item In Allianz-Szenarien dürfen verbündete Spieler frei und jederzeit während ihrer Spielzüge (außer im Kampf) Ressourcen verschenken oder mit Verbündeten handeln.\n"
+"  \\item In kooperativen Szenarien dürfen verbündete Spieler Ressourcen an andere Spiele geben, wenn sie einen Handelsposten besuchen..\n"
```
Comment (qwrtln) [inline translations/trading.tex/de.po:92]:
```suggestion
"  \\item In kooperativen Szenarien dürfen verbündete Spieler Ressourcen an andere Spiele geben, wenn sie einen Handelsposten besuchen.\n"
```

**Comment by DerSchlupf** [comment]
Picture updatet!

File: translations/trading.tex/de.po [hunk 2]
```diff
@@ -78,6 +79,21 @@ msgid ""
...
+"\\begin{itemize}\n"
+"  \\item \\pagetarget{Trading}{Handle}\\index{Handeln} beliebig viele Ressourcen gemäß der Handelskurse laut \\hyperlink{Trade Table}{Übersicht auf dem rückseitigen Cover},\n"
+"  \\item \\textbf{Entferne genau eine Karte} aus deiner Hand und erhalte 1 \\svg{gold},\n"
+"    \\note{5}{Sonderfähigkeiten, Statistiken, die Startfähigkeit und Magische Pfeile können im Handelsposten \\textbf{nicht entfernt werden}.}\\par\n"
+"  \\item Kaufe genau eine \\pagelink{War Machines}{Kriegsmaschine} (benötigt Schutzwall-Erweiterung).\n"
+"\\end{itemize}\n"
+"In \\textbf{Allianz-} und \\textbf{kooperativen} Szenarien gelten folgende erweiterten Regeln zum Handeln:\n"
+"\\begin{itemize}\n"
+"  \\item In Allianz-Szenarien dürfen verbündete Spieler jederzeit während ihrer Spielzüge (außer im Kampf) Ressourcen untereinander verschenken oder handeln.\n"
+"  \\item In kooperativen Szenarien dürfen Spieler, die einen Handelsposten besuchen, (zusätzlich zum regulären Effekt) Ressourcen an verbündete Spieler verschenken.\n"
```
Comment (qwrtln) [inline translations/trading.tex/de.po:92]:
```suggestion
"  \\item In kooperativen Szenarien dürfen Spieler beim Besuch eines Handelspostens (zusätzlich zum regulären Effekt) Ressourcen an verbündete Spieler verschenken.\n"
```
That's one line less.

File: translations/trading.tex/de.po [hunk 3]
```diff
@@ -78,6 +79,21 @@ msgid ""
...
 "\n"
 msgstr ""
+"Der Ort \\pagelink{Trading Post}{Handelsposten} auf der Abenteuerkarte erlaubt dir genau einen der folgenden Effekte auszuführen:\n"
+"\\begin{itemize}\n"
+"  \\item \\pagetarget{Trading}{Handle}\\index{Handeln} beliebig viele Ressourcen gemäß der Handelskurse laut \\hyperlink{Trade Table}{Übersicht auf dem rückseitigen Cover},\n"
+"  \\item \\textbf{Entferne genau eine Karte} aus deiner Hand und erhalte 1 \\svg{gold},\n"
+"    \\note{5}{Sonderfähigkeiten, Statistiken, die Startfähigkeit und Magische Pfeile können im Handelsposten \\textbf{nicht entfernt werden}.}\\par\n"
+"  \\item Kaufe genau eine \\pagelink{War Machines}{Kriegsmaschine} (benötigt Schutzwall-Erweiterung).\n"
+"\\end{itemize}\n"
+"In \\textbf{Allianz-} und \\textbf{kooperativen} Szenarien gelten folgende erweiterten Regeln zum Handeln:\n"
```
Comment (qwrtln) [inline translations/trading.tex/de.po:89]:
```suggestion
"In \\textbf{Allianz-} und \\textbf{kooperativen} Szenarien gelten erweiterte Handelsregeln:\n"
```
That's also one less line, if that's OK?

**Comment by DerSchlupf** [comment]
I've done the changes, but i'm not sure, if its better, because in the first version there was the Point "In kooperativen Szenarien..." completly at the right column and now there is one line left and the rest right.

**Comment by qwrtln** [comment]
> I've done the changes, but i'm not sure, if its better, because in the first version there was the Point "In kooperativen Szenarien..." completly at the right column and now there is one line left and the rest right.

You can still see that you have a lot more text than the original:

de-41

Good candidates for shortening are the first paragraph and the bottom two bullet points. Doing that should fix your layout, and improve the art, too.

**Comment by DerSchlupf** [comment]
New picture! In my opionion I'm ready with the trading section.

**Review by qwrtln — APPROVED** [review]
I would suggest shaving off the final "den.", but the current layout is tolerable.

**Comment by DerSchlupf** [comment]
Its done. I found another solution.

**Pull request merged**